LEGO1/mxpresenter.cpp

```diff
@@ -155,6 +157,63 @@ void MxPresenter::Enable(MxBool p_enable)
   }
 }
 
+// OFFSET: LEGO1 0x100b5310
+char *PresenterNameDispatch(MxDSAction *p_action)
```

Comment:
Maybe we should use a reference instead of a pointer as an argument in this case? We won't really know what the developers used originally since it should compile to the same assembly, so it's up to us to make a decision here. We should probably establish a guideline we want to follow for the decomp.